nb2workflow/nbadapter.py

```diff
@@ -433,12 +434,12 @@ def update_summary(self, **d):
 
         
 
-    def execute(self, parameters, progress_bar = True, log_output = True, inplace=False, tmpdir_key=None, callback_url=None):
+    def execute(self, parameters, progress_bar = True, log_output = True, inplace=False, tmpdir_key=None, context={}):
```

Comment:
Although it's good to keep style change separate from code change, I suggest that if we modify the line, we can fix it's style.

Then mutable default argument is a [common source of surprising and hard to trace issues](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).


```suggestion
    def execute(self, parameters, progress_bar=True, log_output=True, inplace=False, tmpdir_key=None, context=None):

    if context is None:
        context = {}
```